packages/components/src/VirtualizedList/utils/resizable.test.js

```diff
@@ -0,0 +1,325 @@
+			resized: true,
+			resizable: true,
+			width: 180,
+			minWidth: 40,
+		};
+		// when
+		const ret = createColumnWidthProps(params);
+		// then
+		expect(ret).toEqual({
+			width: 180,
+		});
+	});
+});
+
+describe('getColumnWidth', () => {
+	it('should return the column matching the dataKey', () => {
+		// given
+		const dataKey = 'hello';
+		const columnsWidths = [
+			{
+				dataKey: 'hello',
+				width: 100,
+			},
+			{
+				dataKey: 'world',
+				width: 50,
+			},
+		];
+		// when
+		const ret = getColumnWidth(dataKey, columnsWidths);
+		// then
+		expect(ret).toBe(columnsWidths[0]);
+	});
+	it('should return undefined', () => {
+		// given
+		const dataKey = 'something else';
+		const columnsWidths = [
+			{
+				dataKey: 'hello',
+				width: 100,
+			},
+			{
+				dataKey: 'world',
+				width: 50,
+			},
+		];
+		// when
+		const ret = getColumnWidth(dataKey, columnsWidths);
+		// then
+		expect(ret).toEqual(undefined);
+	});
+	it('should return an empty object', () => {
+		// given
+		const dataKey = 'something else';
+		const columnsWidths = {
+			dataKey: 'hello',
+			width: 100,
+		};
+		// when
+		const ret = getColumnWidth(dataKey, columnsWidths);
+		// then
+		expect(ret).toEqual({});
+	});
+});
+
+describe('extractResizableProps', () => {
+	it('should return a new array with widths props values', () => {
+		// given
+		const arrayOfReactElements = [
+			{
+				stuff: 'lots of stuff',
+				props: {
+					dataKey: 'hello',
+					minWidth: undefined,
+					resizable: true,
+					width: 100,
+				},
+			},
+			{
+				stuff: 'lots of stuff',
+				props: {
+					dataKey: 'world',
+					resizable: false,
+				},
+			},
+		];
+		// when
+		const ret = extractResizableProps(arrayOfReactElements);
+		// then
+		expect(ret).toEqual([
+			{ dataKey: 'hello', minWidth: 40, resizable: true, width: 100 },
+			{ dataKey: 'world', minWidth: 40, resizable: false, width: undefined },
+		]);
+	});
+	it('should return an empty array', () => {
+		// given nothing
+		// when
+		const ret = extractResizableProps();
+		// then
+		expect(ret).toEqual([]);
+	});
+});
+
+describe('resizeColumns', () => {
+	it('should enlarge columnTwo and shrink columnThree', () => {
+		// given
+		const deltaX = 10;
+		const columnsWidths = [
+			{
+				dataKey: 'columnOne',
+				width: 70,
+				resizable: true,
+			},
+			{
+				dataKey: 'columnTwo',
+				width: 100,
+				resizable: true,
+			},
+			{
+				dataKey: 'columnThree',
+				width: 130,
+				resizable: true,
+			},
+		];
+		const currentIndex = 1;
+		// when
+		const ret = resizeColumns(deltaX, columnsWidths, currentIndex);
+		// then
+		expect(ret).toEqual([
+			{ dataKey: 'columnOne', resizable: true, resized: false, width: 70 },
+			{ dataKey: 'columnTwo', resizable: true, resized: true, width: 110 },
+			{ dataKey: 'columnThree', resizable: true, resized: true, width: 120 },
+		]);
+	});
+	it('should enlarge columnOne and shrink columnThree, skipping columnTwo which is at minimum width', () => {
```

Comment:
This is not a wanted behavior. When we resize columnOne, columnTwo that is resizable should be enlarged.
Same for next test